### ISSUE TITLE
[7.0.7-35] test suite fails on 32-bit

### ISSUE THREAD

**Issue by remicollet** [ImageMagick/ImageMagick#1148]
```
============================================================================
Testsuite summary for ImageMagick 7.0.7
============================================================================
# TOTAL: 86
# PASS:  85
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See ./test-suite.log
Please report to https://github.com/ImageMagick/ImageMagick/issues
============================================================================
```

And in test-suite.log
```
Line: 160, backgroundColor default (#FFFFFFFFFFFF) is incorrect
Testing throwing and catching exceptions. A program crash or a message
that the exception was not caught indicates a test failure.  A properly
formatted exception message indicates success:
Caught exception, good!:
  "lt-attributes: unable to open image 'foo': No such file or directory @ error/blob.c/OpenBlob/3375"
Line: 1061, pixelColor default (#FFFF00000000FFFF) is not canvas color (#FFFF00000000) as expected
Line: 1349, totalColors is 307200 rather than 2 as expected
3 failures
not ok
FAIL: Magick++/tests/tests.tap 2

```

**Comment by remicollet** [comment]
7.0.7-39 still affected

**Comment by remicollet** [comment]
Seems fixed in 7.0.8-3
